mollie-payments-for-woocommerce/includes/mollie/wc/helper/settings.php

```diff
@@ -630,4 +618,94 @@ protected function warnWoo3xRequiredForKlarna( $content ) {
 		return $content;
 	}
 
+    /**
+     * Get current locale by WordPress
+     *
+     * Default to self::SETTING_LOCALE_DEFAULT_LANGUAGE
+     *
+     * @return string
+     */
+    protected function getCurrentLocale()
+    {
+        $locale = apply_filters(self::FILTER_WPML_CURRENT_LOCALE, get_locale());
```

Comment:
Another example of a value that should ideally be injected into the class.

Reply:
May be later during a refactor, right now less code we change the better.